objectVariable/css/css.css

```diff
@@ -0,0 +1,7 @@
+function fun1(){
```

Comment:
# File specific comment 
this is css file and having javascript code.
Please change the extension of it.

Reply:
done